src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderProgramProfileRepository.kt

```diff
@@ -53,6 +54,31 @@ interface OffenderProgramProfileRepository : JpaRepository<OffenderProgramProfil
   )
   fun findActiveAllocations(prisonId: String, excludeProgramCodes: List<String>, courseActivityId: Long?, pageable: Pageable): Page<Long>
 
+  @Query(
+    value = """
+       select new uk.gov.justice.digital.hmpps.nomisprisonerapi.activities.api.FindSuspendedAllocationsResponse(o.nomsId, ca.courseActivityId, ca.description)
```

Comment:
very minor but I don't think you need the `new uk.gov.justice.digital.hmpps.nomisprisonerapi.activities.api.FindSuspendedAllocationsResponse` if the column names match the entity constructors you are retuning,

See https://github.com/ministryofjustice/hmpps-nomis-prisoner-api/blob/efaa424bddc34931eea5ddd4748f8acac36f0807/src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/AdjudicationIncidentChargeRepository.kt

Reply:
Thanks, that's better.